lib/components/animation_component.dart

```diff
@@ -53,7 +54,9 @@ class AnimationComponent extends PositionComponent {
   @override
   void render(Canvas canvas) {
     prepareCanvas(canvas);
-    animation.getSprite().render(canvas, width: width, height: height);
+    animation
+        .getSprite()
+        .render(canvas, width: width, height: height, overridePaint: paint);
```

Comment:
is a default value not necessary?

Reply:
Nope, Sprite already has a default to handle that.